frontend/components/ArticleItem/index.module.scss

```diff
@@ -3,6 +3,7 @@
 
 .articleItem {
   height: 312px;
+  width: 312px;
```

Comment:
This works for all the resolutions?

Reply:
Yes this works for desktop and larger than desktop, mobile is over ridden by the mobile media query.